src/test/java/subway/acceptance/StationPathSearchAcceptanceTest.java

```diff
@@ -0,0 +1,89 @@
+package subway.acceptance;
+
+import io.restassured.path.json.JsonPath;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.jdbc.Sql;
+import utils.AcceptanceUtils;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import static org.mockito.BDDMockito.given;
+import static utils.AcceptanceUtils.createStationLine;
+import static utils.AcceptanceUtils.createStationLineSection;
+
+@DisplayName("지하철 경로 조회 기능")
+@Sql(scripts = "classpath:reset.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.DEFINED_PORT)
+public class StationPathSearchAcceptanceTest {
+
+    /**
+     * Given 1호선 (종로3가, 종로5가, 동대문, 동묘앞)로 이뤄진 노선을 생성한다
+     * Given 4호선 (혜화, 동대문, 동대문역사문화공원)로 이뤄진 노선을 생성한다
+     * Given 부산2호선 (양산, 남양산)로 이뤄진 노선을 생성한다
+     */
+    Map<String, Long> stationIdByName;
+
+    @BeforeEach
+    void setUp() {
+        //given
+        stationIdByName = AcceptanceUtils.createStationsAndGetStationMap(List.of("혜화", "동대문", "동대문역사문화공원", "종로3가", "종로5가", "동묘앞", "양산", "남양산"));
+
+        final Long line1 = createStationLine("1호선", "blue", stationIdByName.get("종로3가"), stationIdByName.get("종로5가"), BigDecimal.valueOf(3L));
+        createStationLineSection(line1, stationIdByName.get("종로5가"), stationIdByName.get("동대문"), BigDecimal.valueOf(5L));
+        createStationLineSection(line1, stationIdByName.get("동대문"), stationIdByName.get("동묘앞"), BigDecimal.valueOf(5L));
+
+
+        final Long line2 = createStationLine("4호선", "mint", stationIdByName.get("혜화"), stationIdByName.get("동대문"), BigDecimal.ONE);
+        createStationLineSection(line2, stationIdByName.get("동대문"), stationIdByName.get("동대문역사문화공원"), BigDecimal.TEN);
+
+        createStationLine("부산2호선", "red", stationIdByName.get("양산"), stationIdByName.get("남양산"), BigDecimal.TEN);
+    }
+
+    /**
+     * When 종로3가에서 동대문역사문화공원으로 경로 조회를 요청한다
+     * Then 종로3가에서 동대문역사문화공원으로 경로 역의 목록으로 (종로3가, 종로5가, 동대문, 동대문역사문화공원)를 응답한다
+     * Then 전체 경로로 18을 응답한다
+     */
+    @DisplayName("정상적인 지하철 경로 조회")
+    @Test
+    void searchStationPath() {
+        //when
+        final JsonPath response = AcceptanceUtils.searchStationPath(stationIdByName.get("종로3가"), stationIdByName.get("동대문역사문화공원"), HttpStatus.OK);
```

Comment:
테스트의 가독성을 위해서 역 id를 Long 타입으로 선언하면 어떨까요?
```suggestion
        final JsonPath response = searchStationPath(종로3가, 동대문역사문화공원), HttpStatus.OK);
```